kiosk/src/Components/GameList.tsx

```diff
@@ -146,8 +146,8 @@ const GameList: React.FC<IProps> = ({ kiosk, buttonSelected }) => {
             >
                 {kiosk.games.map((game, index) => {
                     return (
-                        <SwiperSlide>
-                            <div className="gameTile" style={{ 
+                        <SwiperSlide key={game.id}>
+                            <div className="gameTile"  style={{
```

Comment:
```suggestion
                            <div className="gameTile" style={{
```

Reply:
I'm getting an error in the console about each mapped element not having a key if this is not here.

Reply:
yeah, i just fixed the whitespace for you! the change is good!

Reply:
Oh, sorry, I completely misread the suggested change.